News/Entities/Models/CommentModel.cs

```diff
@@ -10,3 +12,5 @@ public class CommentModel
 
     public int LikeCount { get; set; }
 }
+//should add created at and updated at? 
+//rename comment to content
```

Comment:
TODO:

Reply:
Where can I save my TODO`s if that out of scope now?